profile/2017-03-09-profile.json

```diff
@@ -0,0 +1,47 @@
+{
+    "resource-manager": {
+        "microsoft.authorization": {
+            "version": "2015-07-01",
+            "types": {
+                "locks":"2015-01-01",
+                "policyassignments": "2015-10-01-preview",
+                "policydefinitions": "2015-10-01-preview"
+            }
+        },
+        "microsoft.compute": {
+            "version": "2016-03-30",
+            "types": {}
+        },
+        "microsoft.keyvault": {
+            "version": "2016-10-01",
+            "types": {}
+        },
+        "microsoft.network": {
+            "version": "2015-06-15",
+            "types": {}
```

Comment:
I would rather be more verbose than less. List out the individual resource types here explicitly.

Reply:
Personally I fine to be less explicit, for the point I made here just after.

Reply:
I expect this map to be used to create the schemas that Visual Studio can consume and aid (intellisense, etc) in authoring templates within the IDE, along with SDKs generation. I want this to be a generic map which could serve multiple uses. In that case, if it is not explicitly verbose, i.e., if all the resource types and corresponding API versions are not listed, I am not sure how useful this is.

Also, if sometime down the line ARM decides to provide an API to retrieve cloud capabilities based on profile versions, this map could serve as a starting manifest or this in itself could become a manifest.

Reply:
Either way is okay with me, people looking to this file for guidance can learn to skip or minimize those sections. For purposes of *excluding* types, though, as discussed below, it may be best to list them all minus the excluded ones.

Reply:
This is Open Issue 1